brave_domains/BUILD.gn

```diff
@@ -6,11 +6,19 @@
 import("//build/buildflag_header.gni")
 import("buildflags.gni")
 
-# TODO(petemill): Default target for this project could instead be some cpp
-# helpers which create domain names for consumers for the correct environment
-# and can vary based on dynamic cli param overrides.
+source_set("brave_domains") {
+  sources = [
+    "service_domains.cc",
+    "service_domains.h",
+  ]
+
+  deps = [
+    ":buildflags",
+    "//base",
+  ]
+}
 
-buildflag_header("brave_domains") {
+buildflag_header("buildflags") {
```

Comment:
we should add visibility for this so people don't use it directly and bypass the command line overrides

Reply:
done